core/services/assets/AssetManager.php

```diff
@@ -166,4 +166,36 @@ public function getStylesheetAssets()
     {
         return $this->assets->getStylesheetAssets();
     }
+
+
+    /**
+     * @return void
+     * @since $VID:$
+     */
+    public function enqueueJsAsset($handle)
+    {
+        if($this->assets->has($handle)){
+            /** @var JavascriptAsset $script */
+            $script = $this->assets->get($handle);
+            if ($script->isRegistered()){
+                wp_enqueue_script($handle);
```

Comment:
I'm puzzled, I thought the changes made in `JavaScriptAsset` and `StyleSheetAsset` meant you could just call something like the following:

```php
$script->enqeueueAsset()
```

Reply:
ya I guess that would be better wouldn't it?
these two methods for enqueueing were added because I thought I was going to need to enqueue things from the manager class but that ended up not being the case.
I still think it will be good to be able to do so form the asset managers, but I like your idea better so will change it